functional_tests/scylla_operator/test_functional.py

```diff
@@ -581,3 +581,13 @@ def test_scylla_yaml_override(db_cluster, scylla_yaml):  # pylint: disable=too-m
 
     with scylla_yaml() as props:
         assert configmap_scylla_yaml_content == props
+
+
+@pytest.mark.readonly
+def test_default_dns_policy(db_cluster: ScyllaPodCluster):
+    pods = db_cluster.k8s_cluster.kubectl("get pods -l scylla/cluster=sct-cluster -o yaml", namespace=SCYLLA_NAMESPACE)
+
+    dns_policy = set(pod["spec"]["dnsPolicy"] for pod in yaml.safe_load(pods.stdout)["items"] if pod)
+
+    assert dns_policy == {'ClusterFirstWithHostNet', }, (
+        f'Expected "ClusterFirstWithHostNet" dnsPolicy on all nodes, actually it is {dns_policy}')
```

Comment:
Strange to see possible error message with a set of strings without relation to pods. 
I think would be better to do the same as in the test with tags checking...

Reply:
changed